app/components/date-time-picker.js

```diff
@@ -1,10 +1,42 @@
 import Component from '@glimmer/component';
 import { action } from "@ember/object";
+import { guidFor } from '@ember/object/internals';
+
+const duetDatePickerLocalization = {
```

Comment:
Hardcoded it like this for now. Should this come from the .yml translation files?

Reply:
ah yes, that seems preferable (we use [ember-intl](https://github.com/ember-intl/ember-intl) which has a `t` helper).

Reply:
Done. I decided to generate the day and month names since that's not something you want to manually configure for each language and the browser has those lists already.